rusoto/signature/Cargo.toml

```diff
@@ -33,19 +33,20 @@ log = "0.4.1"
 md5 = "0.7"
 base64 = "0.12"
 hex = "0.4"
-serde = "1"
+serde_json = "1.0"
+serde = { version = "1.0", features = ["derive"] }
 sha2 = "0.9"
 time = "0.2.11"
 pin-project = "0.4"
 percent-encoding = "2"
 tokio = { version = "0.2", features = ["macros"] }
+chrono = "0.4"
```

Comment:
Perhaps it would be better to follow the example of https://github.com/rusoto/rusoto/pull/1829 and remove default features

Reply:
Yeah, if the optional features aren't needed, it's probably nicer to disable them.